pyproject.toml

```
@@ -72,6 +74,7 @@ pytest-faulthandler = "1.5.0"
 pytest-rerunfailures = "5.0"
 pytest-qt = "3.3.0"
 pytest-twisted = "<=1.13.4"
+poetry-dynamic-versioning = "^0.13.1"
```

Why is this line being added back in?
```suggestion
poetry-dynamic-versioning = "^0.13.1"
```

---

It does seem like without it; this project won't have access to the functionality provided by poetry dynamic versioning.

---

According to [these instructions](https://pypi.org/project/poetry-dynamic-versioning/) on how to configure poetry dynamic versioning, your statement is not correct.